utils/zsh/patches/010-use-libpcre2.patch

```diff
@@ -0,0 +1,638 @@
+--- a/Src/Makefile.in
++++ b/Src/Makefile.in
+@@ -35,7 +35,7 @@ VPATH           = @srcdir@
+ sdir            = @srcdir@
+ sdir_top        = @top_srcdir@
+ INSTALL         = @INSTALL@
+-LN		= @LN@
++LN		= @LN_S@
```

Comment:
I wonder, why do we such change? This is not included in https://sourceforge.net/p/zsh/code/ci/b62e911341c8ec7446378b477c47da4256053dc0/ or if you rather prefer GitHub: https://github.com/zsh-users/zsh/commit/b62e911341c8ec7446378b477c47da4256053dc0

Reply:
Yes, it is a work-around to be able to configure/build on openwrt build system which lacks AC_PROG_LN, which is then on @LN@ - but we do have LN_S which is enough, hard linking is not necessary.

Reply:
Ok, in this case, it seems you are the author of this patch, thus please try to describe it in commit description in the patch itself (you can basically copy&paste what you said)